README.md

```diff
@@ -38,6 +38,21 @@ modules
     'FAV' : applies to only your favorite modules, set in ~/.y2m
     <lowercase string> : applies to the named modules
 
+## Authentication
+
+The tool will work even without GitHub authentication, *unless* you share
+an IP with the rest of your company, in which case it will fail because of
+rate limiting. To get an 83 times better rate, authenticate:
+
+Generate an [access token](https://github.com/settings/tokens)
```

Comment:
Um, which scope for the token is needed? I guess just a `public_repo` is enough in this case, isn't it?

And maybe you could directly link to https://github.com/settings/tokens/new ...